extension/package.json

```diff
@@ -1614,6 +1614,7 @@
     "lint-staged": "13.1.0",
     "mocha": "10.0.0",
     "mock-require": "3.0.3",
+    "process-exists": "4.1.0",
```

Comment:
This is the last non-esm only version. Renovate will complain, we will ignore it.